src/main/scala/viper/server/ViperCoreServer.scala

```diff
@@ -133,7 +132,7 @@ class ViperCoreServer(private var _config: ViperConfig) {
       if (_verificationTask != null && _verificationTask.isAlive) {
         _verificationTask.interrupt()
         _verificationTask.join()
-        println(s"Job #$id has been successfully interrupted1.")
+        println(s"Job #$id has been successfully interrupted.")
```

Comment:
No ```println``` in the final version.

Reply:
These print statements were no added by me. They were already present in the old version of viperser. E.g.

https://github.com/viperproject/viperserver/blob/442a0e4320f8ebdf9911789bdc0143c7f6cfb823/src/main/scala/viper/server/ViperServer.scala#L209

There's plenty of them throughout core server. Should I remove them all. Should I replace them with logging statement?